airflow/operators/slack_operator.py

```diff
@@ -8,26 +9,49 @@ class SlackAPIOperator(BaseOperator):
     The SlackAPIPostOperator is derived from this operator.
     In the future additional Slack API Operators will be derived from this class as well
 
-    :param token: Slack api token
-    :type token: String
+    :param token: Slack API token (https://api.slack.com/web)
+    :type token: string
     :param method: The Slack API Method to Call (https://api.slack.com/methods)
-    :type method: String
-    :param param: API Method call parameters (https://api.slack.com/methods)
-    :type param: dict
+    :type method: string
+    :param api_call_params: API Method call parameters (https://api.slack.com/methods)
+    :type api_call_params: dict
     """
+
+    @apply_defaults
     def __init__(self,
                  token='unset',
                  method='unset',
-                 params=None,
+                 api_call_params=None,
```

Comment:
Yeah this break the API, not sure how many people are using it, but that would break any call referencing `params`. If you really want to change the name, we should keeps params and warn about upcoming deprecation.